diesel_tests/tests/schema_dsl/structures.rs

```diff
@@ -84,7 +84,7 @@ where
 {
     fn walk_ast(&self, mut out: AstPass<DB>) -> QueryResult<()> {
         out.unsafe_to_cache_prepared();
-        out.push_sql("CREATE TABLE ");
+        out.push_sql("CREATE TABLE IF NOT EXISTS ");
```

Comment:
@weiznich 
I think this is the best way to prevent failing tests for cases where tables already exist. If there's something wrong with this, I'd like to know what it is.

Reply:
That should work :+1: